Include/Acidanthera/Protocol/AudioIo.h

```diff
@@ -28,13 +28,15 @@
 #include <Uefi.h>
 
 /**
-  Audio I/O protocol GUID.
+  Audio I/O 2 protocol GUID - versioned.
```

Comment:
The file does not change to AudioIo2, so it is already a breaking change. Just swap the GUID in my opinion and leave the name untouched.